common-api/src/main/java/org/pmiops/workbench/db/model/DemographicSurveyEnum.java

```diff
@@ -54,8 +54,8 @@
 
   private static final BiMap<Gender, Short> CLIENT_TO_STORAGE_GENDER =
       ImmutableBiMap.<Gender, Short>builder()
-          .put(Gender.MALE, (short) 1)
-          .put(Gender.FEMALE, (short) 2)
+          .put(Gender.MAN, (short) 1)
```

Comment:
Should we burn the old 1 & 2 values and make new ones for MAN and WOMAN? Or are they really exact replacements?

Reply:
Its the exact replacement, i can still have this as MALE AND FEMALE and have ui label MAN and WOMAN , but since we are going to reintroduce the table itself i thought its best that our enum matches the label